src/include/duckdb/execution/join_hashtable.hpp

```diff
@@ -168,18 +173,18 @@ class JoinHashTable {
 	//! The join type of the HT
 	JoinType join_type;
 	//! Whether or not the HT has been finalized
-	bool finalized;
+	bool finalized {false};
```

Comment:
Are these initializers required?

Reply:
No, it is just good practice, but I can remove it.